config/storybook-preset-carbon/index.js

```diff
@@ -35,6 +37,36 @@ module.exports = {
   ].filter(Boolean),
 
   webpack(config) {
+    const sassLoader = {
+      loader: 'sass-loader',
+      options: {
+        prependData() {
+          return `
+            $feature-flags: (
+              ui-shell: true,
+              enable-css-custom-properties: ${CARBON_REACT_STORYBOOK_USE_CUSTOM_PROPERTIES},
+            );
+          `;
+        },
+        sassOptions: {
+          includePaths: [path.resolve(__dirname, '..', 'node_modules')],
```

Comment:
Might help to include `implementation: require('sass')` here to make sure it will use dart sass